### DISCUSSION TITLE
TimescaleDB support

### DISCUSSION THREAD

**Discussion by galleon** [mindsdb/mindsdb#2904] [Get help] [answered]
Hi

I am new to mindsdb and what wondering if it would support Timescaledb since it supports postgres (timescaledb being build on top of it)?

Thanks
Guillaume

**Comment by ZoranPandovski** [comment, marked as answer]
Hey @galleon we just merged https://github.com/mindsdb/mindsdb/pull/2843 so with the new release by e.o.w we will officially support TimescaleDB.
